src/index.js

```diff
@@ -13,8 +13,19 @@ export { default as Avatar } from "./Avatar/Avatar";
 export { default as AlgoliaSearch } from "./AlgoliaSearch";
 export { default as Badge } from "./Badge";
 export { default as SpinningDots } from "./SpinningDots";
+export {
+  Dropdown,
+  DropdownTrigger,
+  DropdownContent,
+  DropdownLabel,
+  DropdownItem,
+  DropdownSeparator,
+} from "./DropdownMenu";
 
 export { default as Text } from "./Typography/Text";
 export { default as Heading } from "./Typography/Heading";
 export { default as UnorderedList } from "./Typography/UnorderedList";
 export { default as ListItem } from "./Typography/ListItem";
+
+
+export { default as Menu } from "./Menu";
```

Comment:
seems like we don't use the Menu with the changes on https://github.com/Maxihost/control-next/pull/277 so may we remove it directly?

Reply:
Thanks, done in -ui/pull/166/commits/edb5a93fa08faf2ad2ecf75625187f69087296c1

Reply:
Awesome, I think we're good to go with this one 🎉